Index: index.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\n/**\n * Created by PhpStorm.\n * User: extendeal\n * Date: 13/06/18\n * Time: 14:30\n */
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- index.php	(date 1528911274000)
+++ index.php	(date 1528913830000)
@@ -4,4 +4,5 @@
  * User: extendeal
  * Date: 13/06/18
  * Time: 14:30
- */
\ No newline at end of file
+ */
+require __DIR__ . '/vendor/autoload.php';
